Development/pcm_alt.yml

```diff
@@ -0,0 +1,45 @@
+name: ConsIndShock
+
+states:
+  - name: m
+    long_name: market resources
+    domain: [0, .inf]

```

Comment:
`domain` can go under the state description, along with additional metadata that might be useful